CHANGES/952.bugfix

```diff
@@ -0,0 +1 @@
+Make it clear when a user is an SSO user.
```

Comment:
I recommend the PR title (which will likely become the commit title) and change log entry be changed to something like:
`Add auth_provider to users/ endpoint to denote an SSO user`